src/commandHandlers/bio.ts

```diff
@@ -13,8 +13,8 @@ import { log } from '../logger';
 export const command = async (arg: string, embed: MessageEmbed, message: Message) => {
     const args = arg.split('||');
     const mention = message.mentions.users.first();
-
-    if (args[0].length && !mention && !config.BIO.includes(args[0].trim())) {
+    const field = args[0].toLowerCase().trim();
```

Comment:
You will also need to remove it from like `54`, as the same const cannot be declared twice

Btw we have a command `npm run lint` which you can run locally to check the linter is happy before committing the code

Reply:
ohh yeah. I am actually doing it from the github UI.

Reply:
> I am actually doing it from the github UI

Ah ok, no problem, we have GitHub Actions to run it and check, so no problem 🤓

Reply:
ohh I just understood what github actions do. lol. But how do I check it before making my commit

Reply:
To check it before the commit, you would need to have the project locally and run the command above (or have access to **GitHub CodeSpaces**, but this is not available to everyone yet until October)

Reply:
ohh okay

Reply:
mehn, failed again!

Reply:
This one you might have missed, something like `vscode` would have removed it

```
/home/runner/work/EddieBot/EddieBot/src/commandHandlers/bio.ts:54:19
ERROR: 54:1  no-trailing-whitespace  trailing whitespace
```

Reply:
I'd do it locally if this next try fails

Reply:
**CodeSpaces** is going to be a game changer when it comes out 🎉 - issues like this will be a thing of the past 🤓

Reply:
yeah really, check it out now, is it okay?

Reply:
I think it looks good 👍